File: easy/wordToDigit/wordToDigit.rb
File.open(ARGV[0]).each_line do |line|
	words = line.chomp.split(';')
	answer = ""
	words.each {|x|
		case x when 'zero' then ans = '0'
			when 'one' then ans = '1'
			when 'two' then ans = '2'
			when 'three' then ans = '3'
			when 'four' then ans = '4'
			when 'five' then ans = '5'
			when 'six' then ans = '6'
			when 'seven' then ans = '7'
			when 'eight' then ans = '8'
			when 'nine' then ans = '9'
			else ''
		end
		answer += ans
	}
	puts answer
end

File: codeeval/easy/wordToDigit/wordToDigit.cs

using System.IO;
using System;
class wordToDigit
{
    static void Main(string[] args)
    {
        //using (StreamReader sr = new StreamReader("test_file.txt"))
        using (StreamReader sr = File.OpenText(args[0]))
        {
            string line;
            while ((line = sr.ReadLine()) != null)
            {
                string[] words = line.Split(';');
                string outstring = "";

                foreach (string word in words)
                {
                    switch (word)
                    {
                        case "zero":
                            outstring = outstring + "0";
                            break;
                        case "one":
                            outstring = outstring + "1";
                            break;
                        case "two":
                            outstring = outstring + "2";
                            break;
                        case "three":
                            outstring = outstring + "3";
                            break;
                        case "four":
                            outstring = outstring + "4";
                            break;
                        case "five":
                            outstring = outstring + "5";
                            break;
                        case "six":
                            outstring = outstring + "6";
                            break;
                        case "seven":
                            outstring = outstring + "7";
                            break;
                        case "eight":
                            outstring = outstring + "8";
                            break;
                        case "nine":
                            outstring = outstring + "9";
                            break;
                    }
                }

                Console.WriteLine(outstring);
            }
        }
    }
}
